### ISSUE TITLE
Disconnect all peripherals devices problem

### ISSUE THREAD

**Issue by KamiLordus** [libretro/vice-libretro#494]
Hi @sonninnos could you check out Pitfall II - Lost Cavern?
This is the original version (tap) from Activision that does not want to work.
It needs to disconnect all peripherals devices like drive 9, 10, 11 to work and the game must be started via drive 8. True drive emulation must be turned off.
The game only displays the game logo and when the timer reaches 148 the game freezes and should start at this point.
I will add that I managed to run the game on standalone vice, but unfortunately not on vice core.
In the core settings I have "true drive emulation" turned off and the rest of the settings are standard.

Ra 1.16
Vice core 1945343b
[Pitfall II.zip](https://github.com/libretro/vice-libretro/files/12466011/Pitfall.II.zip)

**Comment by sonninnos** [comment]
Ok, with the latest changes the `vicerc` trick is no longer required.

**Comment by KamiLordus** [comment]
If I may ask one more question.
Everything works great except Pitfall and Pitfall II - Lost Caverns.
Previously they also required .vicerc files from:
Printer4=0
Drive8Type=0

This was fixed in:
https://github.com/libretro/vice-libretro/pull/495

When I try to run it now, none of the games load (Pitfall 1 and 2). I also tried the old method with the vicerc file and it doesn't work either.

For a quick test, I'm uploading my games in .tap format
[taps.zip](https://github.com/libretro/vice-libretro/files/14972169/taps.zip)
